prep-javascript-objects/main.js

```diff
@@ -0,0 +1,13 @@
+var person = {
+  firstName: 'Yuliia',
+  lastName: 'Pch',
+  hobby: 'Reading books',
+  job: 'Graphic designer',
+  previousJob: 'Student'
+};
+console.log(person);
+var fullName = 'Yuliia Pch';
+console.log("The person's name is:", fullName);
+console.log("The person's current job is:", person.job);
+console.log("The person's previous job is:", person.previousJob);
```

Comment:
For the job properties, you can add them to your object after the object was already created.

```js
person.job = 'Graphic Designer';
console.log("The person's current job is:", person.job);
```

Reply:
var person = {
  firstName: 'Yuliia',
  lastName: 'Pch',
  hobby: 'Reading books',
  job: 'Graphic designer',
  previousJob: 'Student'
};
console.log(person);
var fullName = person.firstName + ' ' + person.lastName;
console.log("The person's name is", fullName);
console.log("The person's current job is", person.job);
console.log("The person's previous job is", person.previousJob);
console.log('The complete person:', person);



<img width="801" alt="Screen Shot 2022-06-14 at 2 24 43 PM" src="https://user-images.githubusercontent.com/106217355/173692338-0fd3f483-0243-4167-9468-7210827b097d.png">